cropharvest/eo/eo.py

```diff
@@ -324,52 +316,14 @@ def _export_for_polygon(
         kwargs = dict(image=img, region=polygon, filename=filename, description=description)
         if self.dest_bucket:
             kwargs["dest_bucket"] = self.dest_bucket
+        elif test:
+            kwargs["drive_folder"] = self.test_output_folder_name
         else:
-            kwargs["drive_folder"] = drive_folder
+            kwargs["drive_folder"] = self.cur_output_folder
+
         self._export(**kwargs)
         return True
 
-    @staticmethod
-    def metre_per_degree(lat: float) -> Tuple[float, float]:
-        # https://gis.stackexchange.com/questions/75528/understanding-terms-in
-        # -length-of-degree-formula
-        # see the link above to explain the magic numbers
-        m_per_degree_lat = (
-            111132.954
-            + (-559.822 * cos(radians(2.0 * lat)))
-            + (1.175 * cos(radians(4.0 * lat)))
-            + (-0.0023 * cos(radians(6 * lat)))
-        )
-        m_per_degree_lon = (
-            (111412.84 * cos(radians(lat)))
-            + (-93.5 * cos(radians(3 * lat)))
-            + (0.118 * cos(radians(5 * lat)))
-        )
-
-        return m_per_degree_lat, m_per_degree_lon
-
-    @classmethod
-    def _bounding_box_from_centre(
-        cls, mid_lat: float, mid_lon: float, surrounding_metres: Union[int, Tuple[int, int]]
-    ) -> Tuple[Tuple[float, float, float, float], ee.Geometry.Polygon]:
-
-        m_per_deg_lat, m_per_deg_lon = cls.metre_per_degree(mid_lat)
-
-        if isinstance(surrounding_metres, int):
-            surrounding_metres = (surrounding_metres, surrounding_metres)
-
-        surrounding_lat, surrounding_lon = surrounding_metres
-
-        deg_lat = surrounding_lat / m_per_deg_lat
-        deg_lon = surrounding_lon / m_per_deg_lon
-
-        max_lat, min_lat = mid_lat + deg_lat, mid_lat - deg_lat
-        max_lon, min_lon = mid_lon + deg_lon, mid_lon - deg_lon
-
-        return (min_lon, min_lat, max_lon, max_lat), ee.Geometry.Polygon(
-            [[[min_lon, min_lat], [min_lon, max_lat], [max_lon, max_lat], [max_lon, min_lat]]]
-        )
```

Comment:
Refactored into ee_boundingbox to separate building the ee polygons from this code

Reply:
nice this is much nicer. Thank you